schedule/sles4sap/minimalvm/create_hdd.yaml

```diff
@@ -1,41 +1,40 @@
 ---
-description: 'MinimalVM SAP HDD creation for SLES 16. Maintainer: qe-sap.'
-name: 'minimalvm-sap-createhdd@64bit-virtio-vga ( qemu )'
+description: "MinimalVM SAP HDD creation for SLES 16. Maintainer: qe-sap."
+name: "minimalvm-sap-createhdd@64bit-virtio-vga ( qemu )"
 conditional_schedule:
-    bootloader:
-        MACHINE:
-            'svirt-xen-pv':
-                - installation/bootloader_svirt
-            'svirt-xen-hvm':
-                - installation/bootloader_svirt
-                - installation/bootloader_uefi
-            'svirt-hyperv-uefi':
-                - installation/bootloader_hyperv
-            'svirt-hyperv':
-                - installation/bootloader_hyperv
-                - installation/bootloader_uefi
-            'svirt-vmware70':
-                - installation/bootloader_svirt
-                - installation/bootloader_uefi
-            'uefi-virtio-vga':
-                - installation/bootloader
-            '64bit-virtio-vga':
-                - installation/bootloader_uefi
-    wizard:
-        FIRST_BOOT_CONFIG:
-            'wizard':
-                - jeos/firstrun
-            'cloud-init':
-                - installation/first_boot
+  bootloader:
+    MACHINE:
+      "svirt-xen-pv":
+        - installation/bootloader_svirt
+      "svirt-xen-hvm":
+        - installation/bootloader_svirt
+        - installation/bootloader_uefi
+      "svirt-hyperv-uefi":
+        - installation/bootloader_hyperv
+      "svirt-hyperv":
+        - installation/bootloader_hyperv
+        - installation/bootloader_uefi
+      "svirt-vmware70":
+        - installation/bootloader_svirt
+        - installation/bootloader_uefi
+      "uefi-virtio-vga":
+        - installation/bootloader
+      "64bit-virtio-vga":
+        - installation/bootloader_uefi
+  wizard:
+    FIRST_BOOT_CONFIG:
+      "wizard":
+        - jeos/firstrun
+      "cloud-init":
+        - installation/first_boot
 schedule:
-    - '{{bootloader}}'
-    - '{{wizard}}'
-    - console/journal_check
-    - console/suseconnect_scc
-    - sles4sap/patterns
```

Comment:
so "`pattern` has been removed" is the main difference. Right?

Reply:
correct, see discussion on https://suse.slack.com/archives/C0369JZFBKK/p1746795917152719